app/jobs/hearings/send_sent_status_email.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+class Hearings::SendSentStatusEmail
+
+  def initialize(sent_hearing_admin_email_event:)
+    @sent_hearing_admin_email_event = sent_hearing_admin_email_event
+    @sent_hearing_email_event = sent_hearing_admin_email_event.sent_hearing_email_event
+  end
+
+  def call
+    if email_should_send
+      message = send_email
+      external_message_id = get_external_message_id(message)
+      @sent_hearing_admin_email_event.update(external_message_id: external_message_id)
+      log("sent admin email")
+    end
+  end
+
+  private
+
+  # Once you reach this, the email should be ready to send. If you need to block
+  # emails from sending, add the conditionals to email_should_send or a sub method
+  #
+  # See app/jobs/hearings/send_email.rb::send_email for notes on deliver_now!
+  def send_email
+    email = HearingEmailStatusMailer.notification(
+      sent_hearing_email_event: @sent_hearing_email_event
+    )
+    message = email.deliver_now!
+    message
+  end
```

Comment:
following example from `send_email.rb` so that the job can continue running

```suggestion
  rescue StandardError, Savon::Error, BGS::ShareError => error
    Raven.capture_exception(error)

    log("failure to send email")
    nil
  end
```

Reply:
Ah, thanks. Added that rescue.